### PR TITLE
Fixed the addMedia issue pre MediaRequest API call

### PR DESCRIPTION
**Pull request by emanuallan**
Arun will be reviewing this and looking further into the second portion (post MediaRequest API call). 

Fixes [SPARK-248600](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-248600)

### CODE REVIEW

**Comment by emanuallan** [comment]
Just for clarification, this is not ready to be merged or reviewed-- Work still in progress

**Comment by emanuallan** [comment]
Ready for review!

File: packages/node_modules/@webex/plugin-meetings/src/meeting/request.js [hunk 1]
```diff
@@ -305,6 +305,25 @@ export default class MeetingRequest extends StatelessWebexPlugin {
...
 
+  /**
+   * Clean up function for Media Request after addMedia fails
+   * @param {Object} options options for clean up media request
+   * @param {String} options.locusUrl Locus url
+   * @param {String} options.locusId Locus Id
+   * @param {String} options.selfId self Id
+   * @param {String} options.mediaId Media Id
+   * @returns {Promise}
+   */
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/request.js:316]: can we remove this as we dont need it

File: packages/node_modules/@webex/plugin-meetings/src/constants.js [hunk 2]
```diff
@@ -565,7 +565,8 @@ export const HTTP_VERBS = {
   PUT: 'PUT',
   POST: 'POST',
   GET: 'GET',
-  PATCH: 'PATCH'
+  PATCH: 'PATCH',
+  DELETE: 'DELETE'
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/constants.js:569]: delete this as well

**Review by arun3528 — APPROVED** [review]
Pr looks good once you remove the delete function

**Pull request merged**